cmd/sponge/disk/disk.go

```diff
@@ -10,26 +10,26 @@ import (
 	"github.com/coralproject/shelf/internal/sponge/item"
 )
 
-// LoadItems serializes the content of Items from a file using the
+// LoadItem serializes the content of Items from a file using the
 // given file path. Returns the serialized Item value.
-func LoadItems(context interface{}, path string) ([]item.Item, error) {
-	log.Dev(context, "LoadItems", "Started : File %s", path)
+func LoadItem(context interface{}, path string) (item.Item, error) {
```

Comment:
I've usually used the style that if a function is returning a single value (not a slice) and an error that the single item should be a pointer. That way we get runtime catching (panic's due to nil pointer access) in the event that someone ignores the error incorrectly.